Cargo.toml

```diff
@@ -25,7 +25,8 @@ rustdoc-args = ["--cfg", "feature=\"docs\""]
 [features]
 default = ["hyper-server"]
 hyper-server = ["http-service-hyper"]
-docs = ["unstable"]
+docs = ["unstable", "cookies"]
+cookies = ["cookie"]
```

Comment:
We shouldn't feature gate `cookies`; the dependency itself is small, and using cookies is common for many applications.